packages/trader/src/Stores/Modules/Trading/trade-store.js

```diff
@@ -421,7 +423,17 @@ export default class TradeStore extends BaseStore {
                         type: response.msg_type,
                         ...response.error,
                     };
-                    this.root_store.ui.toggleServicesErrorModal(true);
+                    
+                    // check if token is invalid
+                    if (response.error.code === 'InvalidToken') {
+                    // send logout ,clear contracts, clean up client and then show message
+                        this.root_store.client.cleanUp();
+                        requestLogout().then(() => {
+                            this.root_store.modules.trade.clearContracts();
+                            this.root_store.ui.toggleServicesErrorModal(true);
+                        });
+                    } else this.root_store.ui.toggleServicesErrorModal(true); // for other senarioss
```

Comment:
please also use `{ \n }` format for else to be consistent.

Reply:
typo in `senarioss`

Reply:
Thanks done